test/benchmarks/perftest.pl

```diff
@@ -85,8 +86,8 @@
 
 if(!$is_baseline)
 {
-    open(IN,"$basefile") or die "Couldn't open $basefile for reading.";
-    while(<IN>)
+    open my $IN, '<', $basefile or die "Couldn't open $basefile for reading.";
```

Comment:
Good point. But still, just another fact to remember about Perl (and clear from the context in this case, I think).